docs/source/API/core/utilities/all.md

```diff
@@ -0,0 +1,25 @@
+(KokkosALL)=
+# `Kokkos::ALL`
+
+Defined in header `<Kokkos_Core.hpp>`
+
+```c++
+namespace Kokkos{
+  KokkosInternalType ALL = impl_detail;
+}
+```
+
+Type used as tag to specify the selection of all elements in a dimension.
+
+
+## Examples
+
+```c++
+Kokkos::View<double***[5]> a("A",N0,N1,N2);
+
+auto s  = Kokkos::subview(a,
+              std::pair<int,int>(3,15),
+              5,
+              Kokkos::ALL,
+              Kokkos::ALL);
```

Comment:
I think it is unnecessarily complicated I would have gone for a rank-2 view but it is your call

Reply:
I changed it in [bf8683](https://github.com/kokkos/kokkos-core-wiki/pull/132/commits/bf8683e0a822244b42515488c1f4a58afd9736e0) to 3 dims and ditched the `std::pair`. This way we have a `Kokkos::ALL` for one runtime and one compile-time dimension while still giving a reasonable use of `subview`.